node/clients/sellers.ts

```diff
@@ -20,6 +35,98 @@ export default class SellersClient extends JanusClient {
   }
 
   public async getSellers(): Promise<{ items: Seller[] }> {
-    return this.http.get(SELLERS_PATH)
+    return this.http.get(SELLERS_PATH, {
+      metric: 'sellers-get',
+    })
+  }
+
+  public async getSellersPaginated(
+    opts?: GetSellersOpts
+  ): Promise<GetSellersResponse> {
+    const INITIAL_LIST_OPTIONS = {
+      page: 1,
+      pageSize: 100,
+    }
+
+    const argsWithDefaults = {
+      ...INITIAL_LIST_OPTIONS,
+      ...opts,
+    }
+
+    const { pageSize } = argsWithDefaults
+
+    const page = Math.max(1, argsWithDefaults.page)
+
+    const from = (page - 1) * pageSize
+    const to = page * pageSize
+
+    if (from >= to) {
+      throw new Error(
+        'Invalid pagination values: `from` should be less than `to`. Please make sure you are passing valid values for `page` and `pageSize`.'
+      )
+    }
```

Comment:
now that I think about it, we only need to make sure `page` and `pageSize` are both `> 0`. If that is the case, `from` will always be smaller than `to`. Makes sense?
If that is the case, we can simplify this to:
```suggestion
    const { page, pageSize } = argsWithDefaults

    if (page < 1 || pageSize < 1) {
      throw new Error(
        'Please make sure you are passing valid values for `page` and `pageSize`.'
      )
    }

    const from = (page - 1) * pageSize
    const to = page * pageSize
```

what do you think?

Reply:
Great, makes sense!